tests/rustdoc-json/traits/is_object_safe.rs

```diff
@@ -0,0 +1,19 @@
+#![no_std]
+
+// @has "$.index[*][?(@.name=='FooUnsafe')]"
+// @has "$.index[*][?(@.name=='FooUnsafe')].inner.trait.is_object_safe" false
```

Comment:
```suggestion
// @is "$.index[*][?(@.name=='FooUnsafe')].inner.trait.is_object_safe" false
```

That way you get a better error message from jsondocck, (and also means this test will fail if this ever somehow mathces to `[true, false]`).

The top assert is redundant, but it's fine to keep or remove at your discression,